src/asl.ts

```diff
@@ -1714,11 +1682,55 @@ export class ASL {
             },
           };
         });
+      } else if (expr.op === "=") {
+        if (!(isVariableReference(expr.left) || isIdentifier(expr.left))) {
```

Comment:
doesn't `isVariableReference` return `true` for an `Identifier`?

Reply:
weird, apparently I added the `isIdentifier` in an earlier PR. I guess we'll see if anything fails.

Reply:
Or check the implementation of `isVariableReference`? lol

Reply:
yeah, it looked fine, 🤷 removed in new revision